docs/pages/components/side-navigation.md

```diff
@@ -373,13 +383,11 @@ The lists in both sections (Main and Utility) should have the `fd-nested-list--c
 
 ## Side navigation with icons - condensed state
 Use the `fd-side-nav--condensed` modifier class for `condensed` state.
+Group headers are forbidden in `condensed` state.
```

Comment:
I think we can replace this text with:
The following requirements must be met:
- all first level items of the Side Navigation should have icons
- the Side Navigation should not have Group Headers

@Vanessa-Cusmich

Reply:
agreed

Reply:
Agreed +1

Reply:
Descriptions are changed.